src/pages/UserPage/UserPage.tsx

```diff
@@ -124,6 +167,41 @@ const UserPage: FC = () => {
     [deactivationDate, userId, deactivateUser, t, name, setError]
   )
 
+  const onActivateSubmit: SubmitHandler<FormValues> = useCallback(
+    async (values) => {
+      const { user_activation_notification } = values
+
+      const roleIdsArray = roleOptions
+        .filter((obj) => values?.user_activation_roles?.includes(obj.label))
+        .map((obj) => obj?.id)
+
+      const payload = {
+        userId: userId || '',
+        notify_user: user_activation_notification || false,
+        roles: roleIdsArray || [],
```

Comment:
**[non-blocker]**

It would be good to name the input fields with the same names that BE expects (so in this case it would be institution_user_id, notify_user, roles) instead of (userId, user_activation_notification, user_activation_roles).

Not strictly necessary, just we can avoid a lot of renaming of variables this way + will be easier to search for the correct places in the codebase, if needed.